### PR TITLE
Work around #18

### PR DESCRIPTION
**Pull request by im-kulikov**
Проделанное описано здесь: https://github.com/bem/do-it-yourself-workshop/issues/18#issuecomment-72346598

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# IDE
+.idea
+/*.sublime-project
```
Comment (zxqfox) [inline .gitignore]: Can you add this to your personal .gitignore?

File: .gitignore [hunk 2]
```diff
@@ -24,4 +28,4 @@ build/Release
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
 libs
-bfs-stub
+bfs-stub
```
Comment (zxqfox) [inline .gitignore]: Please revert back eol here.

File: .gitignore (hunk 1 above)
Comment (im-kulikov) [inline .gitignore]: yep, one moment

File: .gitignore (hunk 2 above)
Comment (im-kulikov) [inline .gitignore]: yep, one moment

**Comment by im-kulikov** [comment]
@zxqfox all done

**Comment by zxqfox** [comment]
Thanks!

:+1: since bem-components `v2.0.0` was release and islands theme is there forever.

**Comment by im-kulikov** [comment]
@zxqfox thnx! nice work

**Comment by dab** [comment]
@im-kulikov круто, благодарю. Сосквошь коммиты и вольем.

**Comment by im-kulikov** [comment]
@dab готово

**Pull request closed without merging**